cpu/pmqa.py.data/pmqa.yaml

```diff
@@ -0,0 +1,7 @@
+#Usage: run_arg can  any of one from below list
```

Comment:
@PraveenPenguin no, you can grep the log if the exit_status does not provide what you want.